app/src/main/java/com/example/keybindhelper/RecyclerViewAdapters/ProjectAdapter.java

```diff
@@ -88,7 +86,7 @@ public void onBindViewHolder(@NonNull ProjectViewHolder holder, int position) {
                     if(CurrentProjectManager.CurrentProject.id==p.id) {
                         CurrentProjectManager.loadFirstProject();
                     }
-                    fragment.RefreshProjectList();
+                    fragment.refreshProjectList();
```

Comment:
the refreshProjectList() was RefreshProjectList(). I had to rename the method per convetions
https://kotlinlang.org/docs/coding-conventions.html#function-names